fs/ftpfs.py

```diff
@@ -390,6 +420,7 @@ def __init__(
         self.timeout = timeout
         self.port = port
         self.proxy = proxy
+        self.tls = tls
 
```

Comment:
PyFilesystem is quite explicit about things most of the time, so I think we should have a hard error here if `tls=True` and `FTP_TLS is None`. Could you please update the code so that it raises `fs.errors.CreateFailed("FTP over TLS not supported")` here?

Reply:
Even better, we could chain the eventual import error into the `CreatedFailed`, by importing first with:
```python
try:
    from ftplib import FTP_TLS
except ImportError as err:
    FTP_TLS = err
```
and then in `FTPFS.__init__`:
```python
if self.tls and isinstance(FTP_TLS, Exception):
    six.raise_from(fs.errors.CreateFailed("FTP over TLS not supported"), FTP_TLS)
```